lms/static/sass/course/layout/_courseware_header.scss

```diff
@@ -59,6 +59,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
This ensures that the navigational bar at the top of the page (with Courseware, etc) doesn't print.